doc/api/tls.md

```diff
@@ -1027,6 +1027,11 @@ added: v0.3.2
     force SSL version 3. The possible values depend on the version of OpenSSL
     installed in the environment and are defined in the constant
     [SSL_METHODS][].
+  * `secureOptions` {number} The options via bitmask affecting the protocol
+    behavior of SSL. This can be used to limit the versions of SSL/TLS, e.q.
+    `crypto.constants.SSL_OP_NO_TLSv1 | crypto.constants.SSL_OP_NO_TLSv1_1` to
+    deny TLSv1 and TLSv1.1 connections. For more details, see
+    [Crypto Constants][].
```

Comment:
There are no more details in the Crypto Constants section! There are actually three sections, and I can't tell which of the options from which of the sections are actually allowed to be specified as a `secureOption`. Probably there should be a specific section for the options that are allowed in `secureOptions`, unless they all are? Or can the options that are valid for `secureOptions` be used in other places?